cypress/support/component.ts

```diff
@@ -0,0 +1,39 @@
+// ***********************************************************
+// This example support/component.ts is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
```

Comment:
Do we need this entire auto-added component example? I guess I'm okay with it, but I wanted to check you know that it's here, just in case it was added accidentally. @gsambrotta

Reply:
It was autogenerated by Cypress when I updated with the CLI. 
I can remove it. Let's keep it clean :)

Reply:
@lukestanley I decided to delete all the support folder. This folder was created automatically by the CLI when I added the option to configure for testing Angular components. But as we do not have any tests for components, the tests were broken. 
I therefore deleted the configuration and thought it was good to have an example there anyway.
One a second thought, I don't think it makes sense. If we decide to create a test for component, we can easily regenerate this folder